test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -731,18 +720,13 @@ def setUp(self):
         self.load_hook(self.hook_file)
```

Comment:
It looks like load_hook() is duplicating the events and the creating, importing and enabling of the hook in the following 6 lines.

Reply:
Thanks for your comments @vccardenas. I have removed the duplicate code and attached the latest execution logs (cgroup_1mom_try3.txt)

Reply:
@anamikau , thanks for making the changes - looks good to me.